actix-web-codegen/src/route.rs

```diff
@@ -85,10 +87,19 @@ impl Args {
                                 "Attribute guard expects literal string!",
                             ));
                         }
+                    } else if nv.path.is_ident("wrap") {
+                        if let syn::Lit::Str(lit) = nv.lit {
+                            wrappers.push(lit.parse()?);
+                        } else {
+                            return Err(syn::Error::new_spanned(
+                                nv.lit,
+                                "Attribute guard expects type",
```

Comment:
I guess we could improve the wording though.

```suggestion
                                "Attribute wrap expects type",
```